### ISSUE TITLE
Revise methodology for battery cost share of BEV and PHEV vehicles

### ISSUE THREAD

**Issue by robbieorvis** [EnergyInnovation/eps-us#267]
Our estimates for battery costs and their contributions to vehicle prices are quite off from other modeling data available, due to a mix of data and methodological issues.

To estimate future battery costs within BEV and PHEV vehicles we do the following:

1) Calculate the change in annual battery costs in the endogenous learning module as a percentage of start year costs (calculated endogenously)
2) Calculate the share of vehicle costs that is from the battery (time-series data input)
3) Calculate future battery share costs based on cost declines from endogenous learning (calculated endogenously)
4) Sum this back with remaining costs to find net future vehicle costs (calculated endogenously)

There are areas where model results do not align with the latest research and projections:

1) Our start year battery cost share of vehicle price is far too high. In the US model, it's showing about $22,000 for the battery in a new passenger LDV in the start year. This compares to $8,000-$11,000 (depending on vehicle type) from the latest Argonne national lab modeling. As another point of comparison, ICCT estimates 2020 pack prices were $150/kWh to produce,. A typical EV has a ~70 kWh pack, so right around $10,500 per vehicle. We are about 2x too high.

2) Prices for batteries and EVs drop too low. The latest data from ICCT, which compiles estimates from others, shows a bottoming out around >$50/kWh by 2035 for battery packs, but we are below that and get as low as $25/kWh by 2050, about half of the most optimistic estimates I've seen.

I would suggest we revise our methodology and data here. 

In the near term, this can be partly addressed by modifying the data for _BBSoEVP BAU Battery Share of Electric Vehicle Price_ so that our battery prices are correct. We will also need to update either our learning rate or our global battery deployment projections to align battery price projections from there. 

In the long run, we should modify the methodology when we implement changes for #205. Here, I would suggest we explicitly track the price of batteries in the model and feed into the transportation sector, based on the average battery size by vehicle and cargo type. From there, we can add the RPE (discussed in the other thread) to get the final impact on price. This will help better calibrate the model in the future to ensure our battery prices are reasonable and to answer questions about battery prices. We can also account for projected changes in battery size. For example, ICCT projects that the average battery size per vehicle will decrease as battery efficiency improves and vehicle weights decrease.

If we are able to explicitly track battery prices this way, we can add a new policy lever, which is a battery manufacturing incentive (like what is in the IRA and other countries are considering).

**Comment by mkmahajan** [comment]
Agree that we should revise our methodology. But just for background, I believe we agreed to our current data approach for battery cost values a while ago when we found that our LDV prices were much higher than ICCT even though we were applying similar learning rates. When we looked at ICCT research, we saw that a significant portion of their projected cost declines were not related to the battery and instead were due to things like economies of scale as EV manufacturing facilities got up and running. So with our current methodology, we needed to make a larger portion of the vehicle price subject to endogenous learning. We'll need to capture the non-battery cost declines separately.

**Comment by robbieorvis** [comment]
We can and should probably separate out the component costs in the future and allow for cost declines for both. You are right they have significant cost declines aside from the battery (though some of that is related to reductions in battery size). Their indirect costs come down a lot. For example, see the email below with Pete Slowik from ICCT regarding EIA's modeling:

The largest cost reduction, both absolute and percent reduction, are battery packs. This is due to both the reduced $/kWh for batteries and also improved vehicle efficiency which enables reduced battery size for the same range. Starting in cells D13 I also provided the battery pack size and efficiency specifications for the representative 300-mile range SUV for 2022, 2030, and 2035. You can see that as efficiency improves greatly from about 0.40 kWh/mile in 2022 to 0.28 kWh/mile in 2030, the size of the battery pack for the same 300-mile range SUV declines from about 119 kWh in 2022 to 82 kWh in 2030. Our projected efficiency improvements are aggressive and optimistic, but they are identical to the efficiency values that CARB applied in its ACC II regulation for 2030-2035. For context, our assumed 2030 efficiency values are slightly better than the most efficient "best in class" (Tesla, etc.) models from 2022. (For more on this, see the paragraph on page 11 in our report<https://theicct.org/wp-content/uploads/2022/10/ev-cost-benefits-2035-oct22.pdf>)

  *   Part of this efficiency improvement is due to weight reduction in the vehicle and battery packs, but there are also several other reasons. EV efficiency can improve due to electric component (battery, motor, power electronic) as well as vehicle-level (mass reduction, aerodynamics, tire rolling resistance) improvements.
  *   If EIA is assuming that a 300-mile range BEV has the same pack size in 2022 and 2030, that's a key difference compared to our study.

The next largest cost reduction is indirect costs. The primary driver for declining indirect EV costs is reduced R&D costs on a per vehicle basis. Although the absolute indirect costs are reduced by several thousand dollars from 2022 to 2030, the ratio of indirect costs relative to direct manufacturing costs don't change that much (i.e., this is the minor difference between our effective RPE of 1.5 in 2022 vs. 1.45 in 2030). As direct costs decline, so do indirect costs.

The third largest cost reduction is in the non-battery powertrain, which we model to be about $700 less in 2030 compared to 2022. Non-battery powertrain components include things like thermal management, electric drive module, high voltage cables, the on board charger, etc. Table 6 in our report summarizes the BEV powertrain components and their costs in 2022 and 2030. The costs for most of them are reduced by a few hundred dollars by 2030.

Finally, we also expect to see relatively smaller cost reductions in the vehicle non-powertrain and assembly. Our BEV non-powertrain and assembly costs in 2020 are initially identical to those of ICEs. In future years, we apply a 5% decrease due to 30% lower cost of assembly, and the fact that assembly comprises about 17% of non-powertrain direct costs. We think there's an opportunity for manufacturers to shift toward dedicated BEV platforms that enable new areas of cost reductions and economies of scale, thus providing some assembly cost reductions. (See discussion at bottom of page 11).

**Comment by mkmahajan** [comment]
I've incorporated Robbie's work from the #267 branch with some tweaks. I've also added in a BAU battery manufacturing subsidy (populated with the IRA tax credit values) and a policy lever for an additional subsidy amount. 

I will leave this issue open for now as I'd like to do some more data calibration to ensure our vehicle prices align with ICCT estimates. But I believe the structural elements are in place. @robbieorvis let's plan to discuss at our next check-in to make sure no additional elements are needed to represent this correctly.

**Comment by mkmahajan** [comment]
I made one additional structural edit to simply the code and make it easier to align vehicle prices with ICCT's trajectories. Now that this issue is finalized, I can go ahead and calibrate the transportation logit parameters and finalize our vehicle sales results for the 4.0 release.
